docs/src/css/custom.css

```
@@ -331,7 +331,7 @@ a code {
   color: var(--code-link-text);
 }
 
-a[href^="https://api.slack.com/methods"] > code {
+a[href^="https://docs.slack.dev/reference/methods"] > code {
   background-color: var(--method-link-background);
   color: var(--method-link-text);
 }
```

```suggestion
a[href^="https://api.slack.com/methods"] > code {
  background-color: var(--method-link-background);
  color: var(--method-link-text);
}

a[href^="https://docs.slack.dev/reference/methods"] > code {
  background-color: var(--method-link-background);
  color: var(--method-link-text);
}
```

 i gave you slightly misleading advice in slack -- as the `href^="/reference/methods"` copy won't pick up anything since it's the tools site. let's add both the old and new please